mxcubeweb/routes/ra.py

```diff
@@ -43,25 +43,27 @@ def handle_timeout_gives_control(sid, timeout=30):
                 return make_response(msg, 409)
 
         current_user.requests_control = True
-        server.user_datastore.commit()
+        app.usermanager.update_user(current_user)
+
+        server.emit("observersChanged", namespace="/hwr")
```

Comment:
`userChanged` doesn't need to be sent here. Instead, the `user` state is now refetched right after calling the endpoint in the front-end, in the `requestControl` thunk.